.github/workflows/release.yml

```diff
@@ -47,3 +49,18 @@ jobs:
             ./dist/*.whl
           prerelease: ${{ contains(github.ref, '-rc') }}
           generate_release_notes: ${{ !contains(github.ref, '-rc') }}
+
+      - name: Publish to Test PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1.5
+        if: ${{ contains(github.ref, '-rc') }}
+        with:
+          repository_url: https://test.pypi.org/legacy/
```

Comment:
I grabbed this configuration from the commit when it was removed: https://github.com/cal-itp/littlepay/pull/11/commits/871ea3f4d7ce4572e8a83a33f259945bc11c1c3c

I noticed that for `hashfields`, the configuration has [`user` and `password`](https://github.com/cal-itp/hashfields/blob/main/.github/workflows/release.yml#L131-L132) fields where PyPI credentials are specified... are they not needed for this repo?

If it turns out we _do_ need to provide a `user` and `password` to the step, we'll need to generate and set those secrets since I don't see any secrets for PyPI in this repo's settings.

Reply:
Oh, I see now it's because this uses `id-token: write` under `permissions` for trusted publisher

Reply:
@angela-tran thanks for noting this difference with `hashfields`.

I've created https://github.com/cal-itp/hashfields/issues/62 to track refactoring that workflow to use Trusted Publishers